src/components/TreeGrid/TreeGrid.tsx

```diff
@@ -260,6 +264,29 @@ export class TreeGrid extends React.PureComponent<ITreeGridProps, ITreeGridState
         );
     }
 
+    private formatBoolCell(cellData: any, rowData: any) {

```

Comment:
copy paste from grid? Can we avoid it somehow?

Reply:
In the first commit it was possible because the method was exposed by QuickGrid interface, but since we are now setting it before mounting of component, that reference is null

Reply:
It can be fixed by moving _getTreeColumnsToDisplay to componentDidMount. Unless theres a reason for it to be in the constructor

Reply:
you could create a boolFormatterFactory(type: BoolFormatType) to capture the type and thus avoid the need for this and _ref.

Reply:
Done!